dandi/files.py

```diff
@@ -518,6 +519,19 @@ def get_validation_errors(
         return errors
 
 
+class VideoAsset(LocalFileAsset):
+    EXTENSIONS: ClassVar[List[str]] = VIDEO_FILE_EXTENSIONS
+
+    def get_metadata(
+        self,
+        digest: Optional[Digest] = None,
+        ignore_errors: bool = True,
+    ) -> BareAsset:
+        metadata = get_default_metadata(self.filepath, digest=digest)
+        metadata.path = self.path
+        return metadata
```

Comment:
why not to move this implementation of `get_metadata` into `LocalFileAsset` to provide such default implementation and thus to avoid duplication across subclasses which just need to use the default?

Reply:
could there be a test added which would ensure that `upload` does recognize video files as "first class citizen"?

Reply:
Done.

Reply:
For testing, what about just checking [here](https://github.com/dandi/dandi-cli/blob/a1a7fd553cf300967adc060302c7e6f6c1b921c8/dandi/tests/test_files.py#L23) that files with one of more of the video extensions are recognized as Dandi files?

Reply:
ok

Reply:
Done.